src/Radio/styles.js

```diff
@@ -16,6 +16,7 @@ export default css`
         height: calc(100% + 2px);
         top: -1px;
         left: -1px;
+        box-sizing: border-box;
```

Comment:
Is this necessary when we load in the [`CSSReset`](https://github.com/dhis2/ui-core/blob/master/src/CssReset/index.js#L15)?

Reply:
The import export app doesn't use the `CSSReset` and either we fix the component there or we make sure that the component can be used with and without the reset css

Reply:
Would adding the CssReset there break things? Might also be a solution?

Reply:
That seems to work fine actually